testhelpers/image_privileges.go

```diff
@@ -12,7 +12,7 @@ type ImagePrivileges struct {
 	writable bool
 }
 
-// Creates a new ImagePrivileges, use "readable" or "writable" to set the properties accordingly.
+// NewImagePrivileges Creates a new ImagePrivileges, use "readable" or "writable" to set the properties accordingly.
```

Comment:
```suggestion
// NewImagePrivileges creates a new ImagePrivileges, use "readable" or "writable" to set the properties accordingly.
```

Reply:
If I'm understanding https://github.com/buildpacks/lifecycle/pull/685/files correctly, we don't actually care that `imageName` is an image name. We just care what permissions we want the image to have when we call `testRegistry.Add(myImage, ih.NewImagePrivileges(readWriteImage)))`.

If that's true why not make `readable` and `writable` constants rather than relying on this fuzzy matching? (Here's a playground showing what I'm thinking: https://play.golang.org/p/tR0h1HFW8P- )

Reply:
Yeah, I like it!

Reply:
I added those changes in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1924767685e509286c1db17a8e2c2f791113a114)